app/assets/stylesheets/govuk_publishing_components/components/_image-card.scss

```diff
@@ -208,6 +208,46 @@
   }
 }
 
+.gem-c-image-card__youtube-video-embed {
+  margin: 0;
+  padding-bottom: 68%;
+}
+
+.gem-c-image-card__youtube-video-embed iframe {
+  z-index: 1;
+}
+
+.gem-c-image-card__youtube-thumbnail-image-container {
+  position: relative;
+
+  &:before {
+    background: url("govuk_publishing_components/youtube-play-icon.png");
+    background: url("govuk_publishing_components/youtube-play-icon.svg"), linear-gradient(transparent, transparent);
+    background-repeat: no-repeat;
+    background-position: 50%;
+    background-size: 75px auto;
+    content: "";
+    height: 100%;
+    left: 0;
+    position: absolute;
+    top: 0;
+    width: 100%;
+    z-index: 1;
+  }
+}
+
+.gem-c-image-card__youtube-thumbnail-image {
+  filter: brightness(.9);
+}
+
+.gem-c-image-card__youtube-thumbnail-container-text {
+  margin-top: 10px;
+}
+
+.gem-c-image-card__youtube-thumbnail-container:focus .gem-c-image-card__youtube-thumbnail-container-text {
+  @include govuk-focused-text;
+}
```

Comment:
Not sure where to raise this so not necessarily relevant to this file. 

This might not be something we have control over, but although getting the video to play using the keyboard is possible it is quite fiddly. You need to tab three times into the video element, and only the second focus state is visible (it's a blue box around the 'More' icon, see below). Is there anything we can do to make this more accessible for keyboard users? I've also yet to try this with a screen reader, which we should also consider - again, appreciate we're working with something embedded from another site, so it may not be possible (although that might raise questions about how we add youtube videos to GOV.UK).

![Screenshot 2023-01-06 at 14 15 59](https://user-images.githubusercontent.com/861310/211029667-629f75ed-4a37-4fb0-849b-e693d45042ef.png)

Reply:
@patrickpatrickpatrick were you able to look into making this more accessible for keyboard users?

Reply:
We can't style or change anything about the embed since it's an iframe. I suppose the only thing I could think of that would make the element more keyboard friendly would be making the iframe unfocusable, surrounding it in a focusable element that listens for keypresses and do the corresponding action for the keypresses on the youtube player. Even then we wouldn't be able to change any of the quite bad focus styles on the player. Alternatively, the embed could be made unfocusable by keyboard and we could display the link that appears when there's no embed.

Reply:
https://accessibility.blog.gov.uk/2020/03/16/why-videos-on-gov-uk-use-the-youtube-video-player/ going from this blogpost YT has been quite extensively tested for accessibility and that's why we went for it over other players

Reply:
The Youtube video on this page - https://www.gov.uk/tour - seems to have slightly better focus states. They're obviously still part of the player (as they're blue outlines rather than our focus styles) but each control has some kind of visible focus state. I'm not sure what the difference between that player and this component is, but I think it's worth investigating.

Reply:
Apparently it's because the video on that page is larger. I guess at the size it's rendering currently Youtube assumes we are viewing it on mobile and so defaults to the mobile styling (which won't include the focus states). Going to see if there's a way to force the better desktop styling.

Reply:
https://github.com/alphagov/govuk_publishing_components/pull/3156/commits/10c14d9825273e7c488b799cf827b38a1a0207fd have made an attempt at using this solution here, at desktop viewport the desktop youtube focus states will now be used

Reply:
Hmm sounds hacky. I have other ideas but they also seem hacky.

Reply:
(sorry that deleted comment was in response to an earlier comment)

Reply:
Latest deployment looks a little broken?

![Screenshot 2023-01-10 at 11 02 18](https://user-images.githubusercontent.com/861310/211534564-3b1efe19-4c70-445a-b1e2-843dd1656fd5.png)

Reply:
Unfortunately I'm quite stumped. I'm able to get to use tablet styles but they aren't really that much better than the mobile styles. I think the only way to get it work would be to increase the width of the youtube video and decrease the size of the text container.

Reply:
Have deployed something with the better focus/keyboard controls, let me know if you think that's OK or if we should think of alternative approach.

Reply:
That's better, well done! Shame the video in question has a blue bit behind the blue focus state for the play button, but that hopefully won't be a widespread problem. Did you have to do something complicated to achieve this?

Reply:
It's a bit complex, yes.

```
.gem-c-image-card__youtube-video-embed iframe {
  z-index: 1;
  transform: scale(0.7);
  width: calc(100% + 42px);
}

.gem-c-image-card__youtube-video-embed.gem-c-govspeak__youtube-video iframe {
  top: -8px;
  left: -16px;
}

.gem-c-image-card__youtube-video-embed {
  transform: scale(1.3);
}
```

Basically here I scale down the iframe by 0.3, add the smallest magic number I can to the width of the iframe and then scale up the container of the iframe by 1.3.